crawler.go

```diff
@@ -185,6 +185,7 @@ func (c *Crawler) newErrorHandler() fetchbot.Handler {
 		c.Logger.WithFields(logrus.Fields{
 			"url":    ctx.Cmd.URL(),
 			"method": ctx.Cmd.Method(),
+			"ip":     res.Header.Get("X-ProxyMesh-IP"),
```

Comment:
This is what I would like to achieve, however I don´t like the fact we are coupling here with an specific proxy.

We need to provide a customizable way of doing this.